### PR TITLE
[Model Zoo] add slowonly_r50_non-local_4x16x1

### PR DESCRIPTION
**Pull request by congee524**

### CODE REVIEW

**Comment by congee524** [comment]
waiting for ckpt url

File: configs/recognition/i3d/i3d_nl_dot_product_r50_32x2x1_100e_kinetics400_rgb.py [hunk 1]
```diff
@@ -5,7 +5,6 @@
 
 # model settings
 model = dict(
-    type='Recognizer3D',
```
Comment (innerlee) [inline configs/recognition/i3d/i3d_nl_dot_product_r50_32x2x1_100e_kinetics400_rgb.py:8]: reason?

Comment (congee524) [reply configs/recognition/i3d/i3d_nl_dot_product_r50_32x2x1_100e_kinetics400_rgb.py:8]: it's inherited from _base_/models/i3d_r50.py. There is no need to specify the model type.

**Comment by congee524** [comment]
This PR is ready

**Review by dreamerlin — APPROVED** [review]

**Review by innerlee — APPROVED** [review]

**Pull request merged**